CenterForOpenScience/ember-osf-web
# [ENG-4013] Barium/deprecation this property fallback

-   Ticket: [ENG-4013]
-   Feature flag: n/a

## Purpose

Eliminate this-property-fallback deprecation errors. Might want to review this commit-by-commit. The removal of ember-get-config added a bunch of noise.

## Summary of Changes

1. Upgrade ember-sortable
2. Fix tests that failed this deprecation
3. Upgrade:
    - ember-cli-showdown
    - tag-input
    - Ember-collapsible-panel
    - Ember-content-placeholders
    - Ember-radio-button
4. Remove ember-get-config
5. Move environment.d.ts
6. Fix tests

## Side Effects

Always possible

## QA Notes

Mostly make sure you can drag and drop contributor order in collections and draft registrations.

[ENG-4013]: https://openscience.atlassian.net/browse/ENG-4013?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
